R/rcodesign/build_tarballs.jl

```diff
@@ -24,35 +23,36 @@ if [[ "${target}" == x86_64-apple-darwin* ]]; then
     cp -ra System "/opt/${target}/${target}/sys-root/."
     popd
 fi
-cd $WORKSPACE/srcdir/PyOxidizer-apple-codesign-*/apple-codesign/
-cargo build --release
-install -Dvm 755 "../target/${rust_target}/release/rcodesign${exeext}" "${bindir}/rcodesign${exeext}"
-install_license ../LICENSE
+
+cd $WORKSPACE/srcdir/apple-platform-rs-apple-codesign-$version/
+cargo build --release --package apple-codesign --target-dir ${WORKSPACE}/tmp
+
+install_license $WORKSPACE/srcdir/apple-platform-rs-apple-codesign-$version/apple-codesign/LICENSE
+install -Dvm 755 "${WORKSPACE}/tmp/${rust_target}/release/rcodesign${exeext}" "${bindir}/rcodesign${exeext}"
 """
 
+script = replace(script, raw"$version" => version)
+
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms()
-# Rust toolchain for i686 Windows is unusable
-filter!(p -> !Sys.iswindows(p) || arch(p) != "i686", platforms)
-# Build for PowerPC fails with
-#     error: failed to run custom build command for `ring v0.16.20`
-#
-#     Caused by:
-#       process didn't exit successfully: `/workspace/srcdir/PyOxidizer-apple-codesign-0.17.0/target/release/build/ring-f26cfe5ece208e2f/build-script-build` (exit status: 101)
-#       --- stderr
-#       thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /opt/x86_64-linux-musl/registry/src/github.com-1ecc6299db9ec823/ring-0.16.20/build.rs:358:10
-filter!(p -> arch(p) != "powerpc64le", platforms)
+platforms = [
+    Platform("x86_64", "linux"; libc="glibc"),
+    Platform("aarch64", "linux"; libc="glibc"),
+    # #Platform("x86_64", "linux"; libc="musl"), # Duplicate mount point: /opt/x86_64-linux-musl/nonce
+    Platform("aarch64", "linux"; libc="musl"),
+    Platform("x86_64", "macos"),
+    Platform("aarch64", "macos"),
+    Platform("x86_64", "windows")
+]
 
 # The products that we will ensure are always built
 products = [
-    ExecutableProduct("rcodesign", :rcodesign),
+    ExecutableProduct("rcodesign", :rcodesign)
```

Comment:
Can you please not remove the trailing comma which was there for a very good reason?
```suggestion
    ExecutableProduct("rcodesign", :rcodesign),
```